tests/model_fields/test_integerfield.py

```diff
@@ -174,6 +175,16 @@ def test_rel_db_type(self):
         rel_db_type = field.rel_db_type(connection)
         self.assertEqual(rel_db_type, self.rel_db_type_class().db_type(connection))
 
+    def test_overflow_and_underflow(self):
+        min_value, max_value = self.backend_range
+
+        instance_underflow = self.model(value=min_value - 1)
+        with self.assertRaises(ValidationError):
+            instance_underflow.full_clean()
+        instance_overflow = self.model(value=max_value + 1)
+        with self.assertRaises(ValidationError):
+            instance_overflow.full_clean()
```

Comment:
`test_overflow_and_underflow` seems unnecessary, it's already tested in `test_backend_range_validation`.